identity/webapp/src/frontend/Registration/Registration.style.ts

```diff
@@ -41,8 +40,6 @@ export const HighlightMessage = styled(Space).attrs({
   border-left: 13px solid ${props => props.theme.color('yellow')};
 `;
 
-export const Checkbox = styled(CheckboxRadio).attrs({ type: 'checkbox' })``;
-
 export const CheckboxLabel = styled.div`
```

Comment:
Just imported it as is in Registration, the styled component not adding anything.